lib/paged_vertical_calendar.dart

```diff
@@ -86,37 +91,127 @@ class PagedVerticalCalendar extends StatefulWidget {
   /// scroll controller for making programmable scroll interactions
   final ScrollController? scrollController;
 
+  /// Language Code String
+  final String? languageCode;
+
+  /// init with this Date. If no startDate is given, it takes DateTime.now().
+  final DateTime? initDate;
```

Comment:
I would call this `initialDate` and add a bit more documentation. something along the lines of `the initial date that should be displayed on the calendar`

Reply:
Thanks for your feedback. I will do that.

Reply:
Sorry for the confusing commits. This is my first push request. ;-) I had problems with VS code. Future commits should be better named.
I have improved the logic for the calculation of the initialDate.